simulator/src/main/java/com/hazelcast/simulator/coordinator/Coordinator.java

```diff
@@ -239,21 +235,26 @@ private void executeAfterCompletion() {
     }
 
     void uploadFiles() {
-        if (isLocal(simulatorProperties) && !simulatorProperties.getHazelcastVersionSpec().equals(OUT_OF_THE_BOX)) {
-            throw new CommandLineExitException("Local mode doesn't support custom Hazelcast versions!");
+        StringBuffer agentPublicIps = new StringBuffer();
```

Comment:
The jit should be smart enough to remove the locks. But I'll change it anway; I always forget which one is the one that should not be used.